library/src/test/java/com/google/android/gms/location/Helper.java

```diff
@@ -0,0 +1,17 @@
+package com.google.android.gms.location;
+
+import androidx.annotation.NonNull;
+
+public class Helper {
```

Comment:
It is only used in one test class, please move it there as private method instead of having separate helper class for it

Reply:
Tests updated, class deleted